tech/languages/java/java-sig.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Java SIG
+page: java
+---
+
+## Java Special Interest Group
+
+[Java SIG](https://fedoraproject.org/wiki/SIGs/Java) is a special interest group within Fedora Project. The group focuses on packaging Java libraries and applications, setting and improving standards for packaging and collaboratively managing bigger changes related to Java in Fedora.
+If you are interested in Java and Fedora, join us!
```

Comment:
paragraph

paragraph

e.g. "if you are inte..." looks like a new paragraph, no?